src/main/java/io/confluent/connect/jdbc/TimestampIncrementingOffset.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2016 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.connect.jdbc;
+
+import java.sql.Timestamp;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TimestampIncrementingOffset {
+  private static final String INCREMENTING_FIELD = "incrementing";
+  private static final String TIMESTAMP_FIELD = "timestamp";
+  private static final String TIMESTAMP_NANOS_FIELD = "timestamp_nanos";
+
+  private final Long incrementingOffset;
+  private final Timestamp timestampOffset;
+
+  public TimestampIncrementingOffset() {
```

Comment:
I think it would be better to just have the one constructor and explicitly init with (null, null) in the one place this is used